inductiva/tasks/task.py

```diff
@@ -401,16 +414,56 @@ def download_outputs(
             warnings.warn("Path already exists, files may be overwritten.")
         output_dir.mkdir(parents=True, exist_ok=True)
 
+        download_message = "Downloading simulation outputs to %s"
+
+        if filenames:
+            if file_server_available:
+                logging.info(download_message, output_dir)
+                data.download_partial_outputs(
+                    remote_filesystem,
+                    download_url,
+                    filenames,
+                    output_dir,
+                )
+            else:
+                logging.error("Partial download is not available.")
+
+            # If the user requested a partial download, the full download
+            # will be skipped.
+            return output_dir
+
         zip_path = output_dir.joinpath("output.zip")
+        logging.info(download_message, zip_path)
+
+        if file_server_available:
```

Comment:
can you maybe just have one if file_server_available condition, rather than here and above?

Reply:
Ok, thanks for checking